lib/word_scheduler.rb

```diff
@@ -0,0 +1,50 @@
+class WordScheduler
+
+  attr_reader :job_id
+
+  def initialize(speed, words, color_option)
+    @speed = speed
+    @words = words
+    @color_option = color_option
+  end
+
+  def interval
+    (60.0/@speed).round(1).to_s + 's'
+  end
+
+  def start_sending
```

Comment:
For consistency, you can rename this method to `start` and make it class method like this:

```ruby
class WordScheduler
  def self.start(speed, words, color_option)
    scheduler = self.new(speed, words, color_option)
    scheduler.start
  end

  def start
    # current implementation
  end
end
```